configuration-reference/job-specification.md

```diff
@@ -23,7 +23,7 @@ The following configurations are supported by Pinot
-| jobType                  | <p>Type of job to execute. The following types are supported</p><ul><li><code>SegmentCreation</code></li><li><code>SegmentTarPush</code></li><li><code>SegmentUriPush</code></li><li><code>SegmentMetadataPush</code></li><li><code>SegmentCreationAndTarPush</code>: When you are using NFS or similar that sits behind a controller and you are unable to externally copy segments there, this pushes the segment payload.</li><li><code>SegmentCreationAndUriPush</code>: When using deepstore, this creates the segment on the deepstore and pushes the URI to the controller, the controller then downloads the segment to extract the metadata from the URI and copies to the final destination.</li><li><code>SegmentCreationAndMetadataPush</code>: When using deepstore, this is the most efficient as the segment payload does not have to pass through the controller.</li></ul><p><strong>Note:</strong> For production environments where Pinot Deep Store is configured, it's recommended to use <strong>SegmentCreationAndMetadataPush</strong></p>                                                                                                                                                                                                                                                         |
+| jobType                  | <p>Type of job to execute. The following types are supported</p><ul><li><code>SegmentCreation</code></li><li><code>SegmentMetadataPush</code></li><li><code>SegmentTarPush</code></li><li><code>SegmentUriPush</code></li><li><code>SegmentCreationAndMetadataPush</code>: When using deepstore, this is the most efficient as the segment payload does not have to pass through the controller.</li><li><code>SegmentCreationAndTarPush</code>: When you are using NFS or similar that sits behind a controller and you are unable to externally copy segments there, this pushes the segment payload.</li><li><code>SegmentCreationAndUriPush</code>: When using deepstore, this creates the segment on the deepstore and pushes the URI to the controller, the controller then downloads the segment to extract the metadata from the URI and copies to the final destination.</li></ul><p><strong>Note:</strong> For production environments where Pinot Deep Store is configured, it's recommended to use <strong>SegmentCreationAndMetadataPush</strong></p>                                                                                                                                                                                                                                                         |
```

Comment:
I think we should tweak the passive note, and clean up the grammar--I added suggestions

Reply:
Edited, thank you!